mssql/introspection.py

```diff
@@ -108,6 +109,9 @@ def get_table_description(self, cursor, table_name, identity_check=True):
         # map pyodbc's cursor.columns to db-api cursor description
         columns = [[c[3], c[4], None, c[6], c[6], c[8], c[10], c[12]] for c in cursor.columns(table=table_name)]
 
+        if not columns:
+            raise DatabaseError("Table does not exist (empty pragma)")
```

Comment:
We'll have to make this message specific to MSSQL. `Pragma` is only used in SQLite.